docs/integrations/microsoft-azure/web-apps.md

```diff
@@ -18,6 +18,16 @@ For more information, see [Azure Web Apps](https://azure.microsoft.com/en-us/ser
 The Azure Web Apps app supports:
 * **Web Server Logging.** Information about HTTP transactions using the [W3C extended log file format](http://msdn.microsoft.com/library/windows/desktop/aa814385.aspx). This is useful when determining overall site metrics such as the number of requests handled or how many requests are from a specific IP address.
```

Comment:
web server logging and application diagnostic logs they are same as http logs , AppLogs and console logs so we can remove them.
lets have separate section for Resource logs and Activity Logs similar to https://help.sumologic.com/docs/integrations/microsoft-azure/azure-storage/#log-and-metric-types 
all the log types coming in diagnostic settings should be under resource logs

Reply:
Updated.